### ISSUE TITLE
CFT doesn't seem to like big CAI files (1G)

### ISSUE THREAD

**Issue by Jluna-sada** [GoogleCloudPlatform/cloud-foundation-toolkit#1043]
I seem to be one of the few running into a problem with larger CAI files 1.17GB. I followed the past [issue](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/937)   below that looked promising but "--concurrency" does seem to be an option in v0.4.1. I did notice there was a "--workers" flag but this is still taking a very long time (days!!).             
 I initially thought running this on a bigger VM would help but it doesnt seem to utilize the full CPU capacity 48cores..

TLDR:
I am looking for guidance for speeding things up, bigger cpu/RAM doesnt do the trick.

@morgante

**Comment by morgante** [comment]
CFT Scorecard is mainly meant as a "quick check" tool. It's not really designed for production workflows.

That being said, we did some work in #937 to improve concurrency. @hussainak Do you have any tips you can share?

**Comment by hussainak** [comment]
Hi @Jluna-sada 
We use concurrency to get around the large files issue. Just make sure you have a appropriately sized VM if you choose to run concurrently.

**Comment by Jluna-sada** [comment]
Hi @hussainak and @morgante thanks for the insight, but --concurrency does not seem to be a flag on v0.4.1
<img width="1700" alt="Screen Shot 2022-01-12 at 8 20 48 AM" src="https://user-images.githubusercontent.com/91563569/149351719-c2f9b599-4d41-4c92-b46e-28e9064a48e9.png">

**Comment by hussainak** [comment]
> Hi @hussainak and @morgante thanks for the insight, but --concurrency does not seem to be a flag on v0.4.1 <img alt="Screen Shot 2022-01-12 at 8 20 48 AM" width="1700" src="https://user-images.githubusercontent.com/91563569/149351719-c2f9b599-4d41-4c92-b46e-28e9064a48e9.png">

Hey mate, you'd need to update to the latest version.

**Comment by Jluna-sada** [comment]
Hi @hussainak v0.4.1 is the latest version and I already tried --workers flag set it to 100 and it did not speed things up nor did it utilize the large VM the binary was running on at all as mentioned on the initial post.

I worked with Google directly on the issue and they were able to provide additional  tooling for this large of a file.

**Comment by hussainak** [comment]
Yea 100 is too large, we have 12 for our environment. Try a lesser number.

**Comment by Jluna-sada** [comment]
Hi @hussainak I have tried 12 and 4 and both were taking days.

**Comment by Jluna-sada** [comment]
I got a work around for now but wanted to bring awareness

**Comment by hussainak** [comment]
That's strange as we haven't seen this happening.. please share your
workaround.
